tests/validation/tests/v3_api/common.py

```diff
@@ -1693,6 +1693,7 @@ def wait_for_app_to_active(client, app_id,
         app_data = client.list_app(id=app_id).data
         application = app_data[0]
     while application.state != "active":
+        timeout = start + 300
```

Comment:
Wasn't this intended for the `wait_for_backup_to_active()`method?

Reply:
Right, this change was for testing another test failure. I'll update PR.